packages/lib/src/components/Table/TableControl.tsx

```diff
@@ -0,0 +1,152 @@
+import React from "react";
```

Comment:
You need to make sure you preserve the original File and folder structure used in spruce when moving this over